web3/trace.py

```diff
@@ -0,0 +1,77 @@
+from web3 import formatters
+from web3.utils.empty import (
+    empty,
+)
+
+class Trace(object):
+    def __init__(self, web3):
+        self.web3 = web3
+
+    defaultAccount = empty
```

Comment:
Looks like this isn't used and can be deleted? (which means the import of `empty` can be, too)